sweep/fee_bumper.go

```diff
@@ -512,17 +513,6 @@ func (t *TxPublisher) broadcast(requestID uint64) (*BumpResult, error) {
 
 	txid := record.tx.TxHash()
 
-	// Subscribe to its confirmation notification.
```

Comment:
Typo in commit message for: 
> sweep: add monitor loop to TxPublisher
Whevenr -> Whenever

Reply:
fixed